src/test/resources/config/application-artemis.yml

```diff
@@ -46,8 +46,6 @@ artemis:
         password: fake-password
         token: fake-token
         url: https://continuous-integration.fake.fake
-        vcs-application-link-name: Fake Application Link Name
-        empty-commit-necessary: true
```

Comment:
I missed one review comment to get to 100, so here is the one to fulfill my urge to complete it. Have a cookie 🍪 :)

Reply:
take two 🍪🍪